client/partials/pie.html

```diff
@@ -0,0 +1,2 @@
+<h2>This is directives pie:</h2>
+<div id="macroPie"></div>
```

Comment:
A directive should be a reusable component - so you'll never want to depends on an `id` in a directive template.

Think of your directive as a new HTML tag that can make D3 pie diagrams.

You'll also want to pass state to the directive in some way, so it doesn't depend on any global state.

You should be able to reuse your `pie` directive for any D3 pie chart and use the directive to draw many different pie charts on a page (pseudo code):

```
<div ng-repeat="dataset in datasets">
  <pie dataset="dataset"></pie>
</div>
```